### PR TITLE
Minor update tfor clarity

### PR DESCRIPTION
**Pull request by tariq-eden**
Minor edits, asking for a review partly to practice the process.

### CODE REVIEW

File: content/handbook/Interim_processes/assembly_facilitation.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The following are some initial process templates for structuring our assemblies.
  * Opening circle 
  * Crew report-backs  
  * Peer Q&A  
+ * Rest break
```
Comment (jadehopepunk) [inline content/handbook/Interim_processes/assembly_facilitation.md:19]: Adding a rest break here is fine, but on a larger issue I'm wondering if this framework is too prescriptive in the middle area, particularly around activities in small groups and reformation of crews. Maybe this structure being always the same works well, or maybe it's more a case of having a "main meeting business" items that are selected to go into each meeting

File: content/handbook/Interim_processes/decision_making_process.md [hunk 2]
```diff
@@ -12,14 +12,17 @@ sidebar:
...
...d formality as appropriate to the group. The key to making consensus work is for everyone to express their needs and viewpoints clearly, and for the group to use this information to find a solution which resolves differences enough to build on common ground. A consensus process typically follow some variation of the following stages: introduction; clarification; open discussion (focusing on expressing different perspectives); exploration (focusing on potential solutions); proposal; amendments; agreement-test; and implementation. It is important to remember that consensus agreements function best when everyone shares a common goal and are willing to work together towards it. Finding a solution usually relies on people being flexible about their preferences in order to meet all the cores needs. This requires a lot of trust. For examples of different consensus decision making processes, see those used by [Seeds of Change](https://www.seedsforchange.org.uk/consensus#flowchart), [[Friends of the Earth Australia](https://www.foe.org.au/consensus_decision_making), and [as articulated in 'Consensus-Oriented Decision-Making' by Tim Hartnett (2011)](https://www.consensusdecisionmaking.org/).
```
Comment (jadehopepunk) [inline content/handbook/Interim_processes/decision_making_process.md:21]: These seems improved, nice work

**Review by jadehopepunk — APPROVED** [review]

File: content/handbook/Interim_processes/assembly_facilitation.md (hunk 1 above)
Comment (tariq-eden) [reply content/handbook/Interim_processes/assembly_facilitation.md:19]: Yeah, fair point. I was just intending for it to be indicative based on our first meeting. I'd expect whoever is facilating to adapt the specifics of each meeting. I'll try and make that clearer in the next round of edits. And/or we can come up with a more general guide.

Comment (tariq-eden) [reply content/handbook/Interim_processes/assembly_facilitation.md:21]: I've updated the file to clarify that these are example processes not a prescriptive framework

**Pull request merged**